messaging/app/src/main/java/com/google/firebase/example/messaging/kotlin/MainActivity.kt

```diff
@@ -130,4 +139,41 @@ class MainActivity : AppCompatActivity() {
     }
     // [END ask_post_notifications]
 
+    // [START get_store_token]
```

Comment:
@andreaowu FYI, this snippet was named `log_reg_token` in your original PR, but that conflicts with another snippet that we already have in this repo:

https://github.com/firebase/snippets-android/blob/067d83afbe16fe078e0b68e511827f8cfcdd6467/messaging/app/src/main/java/com/google/firebase/example/messaging/kotlin/MainActivity.kt#L83

So I renamed it to `get_store_token`.
I also put the function declaration on the snippet itself as I feel it'll make it easier for developers to copy and paste this function. But I have no strong opinions on this one.

Reply:
Sounds good!